docs/howto/serialize_poro.md

```diff
@@ -42,4 +42,32 @@ end
 
 The default serializer would be `MyModelSerializer`.
 
+*IMPORTANT*: There is a surprising behavior (bug) in the current implementation of ActiveModelSerializers::Model that
+prevents an accessor from modifying attributes on the instance.  The fix for this bug
+is a breaking change, so we have made an opt-in configuration.
+
+New applications should set:
+
+```ruby
+ActiveModelSerializers::Model.derive_attributes_from_names_and_fix_accessors
+```
+
+Existing applications can use the fix *and* avoid breaking changes
+by making a superclass for new models. For example:
+
+```ruby
+class SerializablePoro < ActiveModelSerializers::Model
+  derive_attributes_from_names_and_fix_accessors
+end
+```
+
+So that `MyModel` above would inherit from `SerializablePoro`.
+
+`derive_attributes_from_names_and_fix_accessors` prepends the `DeriveAttributesFromNamesAndFixAccessors`
```

Comment:
Needs comment that this will be the default  behavior and the option removed in the next major release